templates/whatson/event_page.html

```diff
@@ -108,9 +102,7 @@ <h3 class="event-meta__title">
                                 <p>{{ page.venue_type }}</p>
```

Comment:
I think this should also be `page.get_venue_type_display`

Reply:
Brill good spot thank you! Updated